central/pruning/pruning.go

```diff
@@ -567,6 +574,46 @@ func (g *garbageCollectorImpl) removeOldReportHistory(config *storage.PrivateCon
 	}
 }
 
+func (g *garbageCollectorImpl) removeOldReportBlobs(config *storage.PrivateConfig) {
+	blobRetentionDays := config.GetReportRetentionConfig().GetDownloadableReportRetentionDays()
+	cutOffTime, err := types.TimestampProto(time.Now().Add(-time.Duration(blobRetentionDays) * 24 * time.Hour))
+	if err != nil {
+		log.Errorf("Failed to determine downloadable report retention %v", err)
+		return
+	}
+
+	query := search.NewQueryBuilder().AddRegexes(search.BlobName, common.ReportBlobRegex).ProtoQuery()
+	blobs, err := g.blobStore.SearchMetadata(pruningCtx, query)
+	if err != nil {
+		log.Errorf("Failed to fetch downloadable report metadata: %v", err)
+		return
+	}
+	// Sort reversely by modification time
+	sort.Slice(blobs, func(i, j int) bool {
+		return blobs[i].GetModifiedTime().Compare(blobs[j].GetModifiedTime()) > 0
+	})
```

Comment:
Can you add sort option to the query? The `modified_time` field is sortable using `Blob Modified On` search label.